src/DSP.jl

```diff
@@ -1,14 +1,23 @@
 ## DSP.jl ##
 
+## === FUNCTIONS == ##
+
 for (T, suff) in ((Float64, "D"), (Float32, ""))
 
+    """
+    Convolution between an input Vector{T} 'X', and a kernel/filter Vector{T} 'K'.
+
+    Returns: Void. Computation result is stored in 'result'.
```

Comment:
Also, the typical julia convention for functions which mutate an object is that they are suffixed with an exclamation mark, and `result` is the first argument, so the signature would be `conv!(result, X, K)`